File: web3rpc/rpc-specs/code-samples/javascript/debug/others/chaindbProperty.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const property = "0xe17d821e9a8a8736b9aea8c2de1f3a4934ac0a2f";

  provider.debug
    .chaindbProperty(property, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();

File: web3rpc/rpc-specs/code-samples/python/debug/others/chaindbProperty.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

string = "0xe17d821e9a8a8736b9aea8c2de1f3a4934ac0a2f"

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.chaindb_property(string)

print(debug_response)
